src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -72,29 +59,22 @@ public override IDisplayResult Display(ListPart listPart, BuildPartDisplayContex
                 );
         }
 
-        private async Task<PagerSlim> GetPagerAsync(IUpdateModel updater, ListPart part)
+        private async Task<PagerSlim> GetPagerAsync(BuildPartDisplayContext context)
```

Comment:
GetPagerSlimAsync

Reply:
> It should be GetPagerSlimAsync and we should also have GetPagerAsync

Changed to `GetPagerSlimAsync`. 
What did you mean by also having `GetPagerAsync`? As a settings option too have the other type of pager?